ako-gateway-api/nodes/gateway_model_rel.go

```diff
@@ -295,7 +296,7 @@ func HTTPRouteToGateway(namespace, name, key string) ([]string, bool) {
 				condition.Type == string(gatewayv1.RouteConditionResolvedRefs)) &&
 				condition.Status == metav1.ConditionFalse {
 				statusIndex += 1
-				continue
+				continue outerLoop
```

Comment:
If RouteConditionResolvedRefs is set to False for one parentRef , it will be set to False for every parentRef, So we need to move out of both the loops